src/figuresClasses.ts

```diff
@@ -1,11 +1,110 @@
-export interface Figure {}
+function roundTo(n: number, decimals: number): number {
+  const factor = Math.pow(10, decimals);
 
-export class Triangle implements Figure {}
+  return Math.floor(n * factor) / factor;
+}
+
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
The `shape` property should be initialized as `shape: 'triangle' = 'triangle';` to ensure TypeScript recognizes it as a constant value of type 'triangle'.